### PR TITLE
Add request not found switch case

### PR DESCRIPTION
**Pull request by poomthiti**
1. add switch case to handle in case no request id is found
2. add img for dark and light mode
Screen Shot 2565-01-25 at 17 19 53
Screen Shot 2565-01-25 at 17 20 01

### CODE REVIEW

File: src/pages/RequestIndexPage.re [hunk 1]
```diff
@@ -179,625 +190,666 @@ let make = (~reqID) => {
...
-              </Col>
-              <Col col=Col.Eight>
-                {switch (requestSub) {
-                 | Data({requester}) =>
-                   switch (requester) {
-                   | Some(requester') =>
-                     <div className=Styles.addressContainer>
-                       <AddressRender address=requester' position=AddressRender.Subtitle />
+  switch (requestSub) {
+  | NoData =>
```
Comment (evilpeach) [inline src/pages/RequestIndexPage.re:194]: Let's change to this one
```
| NoData
| Error(_) => ...
```

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: src/pages/RequestIndexPage.re (hunk 1 above)
Comment (poomthiti) [reply src/pages/RequestIndexPage.re]: Done kub

**Comment by pzshine** [comment]
image

image


Can you adjust these 2 images to be same width (120x120)?

**Comment by poomthiti** [comment]
> image
> 
> image
> 
> Can you adjust these 2 images to be same width (120x120)?

done krub, please kindly give it a look

**Review by babybunn — APPROVED** [review]

**Review by babybunn — APPROVED** [review]

**Review by pzshine — APPROVED** [review]

**Review by babybunn — APPROVED** [review]

**Review by babybunn — APPROVED** [review]

**Pull request merged**